packages/vm/src/runTx.ts

```diff
@@ -52,6 +51,8 @@ import type {
 const debug = debugDefault('vm:tx')
 const debugGas = debugDefault('vm:tx:gas')
 
+const DEFAULT_HEADER = createBlockHeader()
```

Comment:
Doesn't this import the block header in runTx? Why do we need this? Could we not instantiate this with the appropriate fields here, so we don't import the whole BlockHeader class? The header needs: `coinbase`, `baseFeePerGas`, `gasLimit`, `getBlobGasPrice()`, `gasUsed` (can mock this one in another PR if wanted)